config/locales/blacklight.de.yml

```diff
@@ -184,6 +184,7 @@ de:
           index: 'A-Z Ordnen'
         count: '%{number}'
         more: 'mehr »'
+        more_html: 'mehr <span class="sr-only">%{field_name}</span> »'
         selected:
```

Comment:
is plain old `more` i18n value un-used now, in favor of `more_html`?  Should it be removed? Or is there a reason to leave it in for backwards compat?  Or just ignore me if you aren't sure, not a blocker of course.